### PR TITLE
Fixed announcement tab regression (bug 1207599)

### PR DESCRIPTION
**Pull request by Nolski**

### CODE REVIEW

**Comment by kumar303** [comment]
r-, this needs a test. Please add a test for all new logic, especially when fixing a regression.

**Comment by Nolski** [comment]
@kumar303 I added a new test for the new logic created.

File: apps/editors/tests/test_views.py [hunk 1]
```diff
@@ -2733,6 +2733,13 @@ def test_form_errors(self):
         doc = pq(r.content)
         eq_(doc('#editor-motd .errorlist').text(), 'This field is required.')
 
+    def test_motd_tab(self):
+        self.login_as_admin()
+        r = self.client.get(self.get_url())
+        announcement_tab = pq(r.content)(
+            'li.top:nth-child(5) > a:nth-child(1)').text()
```
Comment (kumar303) [inline apps/editors/tests/test_views.py]: this test might break when the layout gets shifted around. A better approach would be to add an id to the Announcement anchor and just check that it's visible. But this test is good for now, we can always fix it up later if it becomes a burden.

**Comment by kumar303** [comment]
this will work for now, thanks

**Pull request merged**